kolibri/core/discovery/test/test_network_utils.py

```diff
@@ -23,6 +29,21 @@ def test_valid_ipv4_address(self):
             ],
         )
 
+    def test_valid_ipv4_address__dedupe_port(self):
+        urls = get_normalized_url_variations("192.168.0.1:8080")
+        self.assertEqual(
+            urls,
+            [
+                "http://192.168.0.1:8080/",
+                "http://192.168.0.1/",
+                "http://192.168.0.1:8008/",
+                "http://192.168.0.1:8000/",
+                "http://192.168.0.1:5000/",
+                "https://192.168.0.1:8080/",
+                "https://192.168.0.1/",
+            ],
+        )
+
```

Comment:
If the passed in URL already had a port, it would produce two URLs with the same port if the port was one of the standard ports. This test ensures we dedupe the url variations.